models/post.js

```diff
@@ -0,0 +1,37 @@
+const db = require('../db');
+
+class Post{
+  static create(userId,communityId,content,date,hour){
+    const queryText = 'INSERT INTO posts (user_id, community_id, content, time_made) VALUES ($1, $2, $3, $4);'
+    return db.query(queryText,[userId,communityId,content,(date,hour)])
+  }
+
+  static delete(postId){
+    const queryText = 'DELETE FROM posts WHERE id = $1;';
+    return db.query(queryText, [postId]);
+  }
+
+  static update(content,postId){
+    const queryText = 'UPDATE posts SET content = $1 WHERE id = $2'
+    return db.query(queryText, [content,postId])
+  }
+
+  static read(postId){
+    const queryText = 'SELECT * FROM posts WHERE id = $1;';
+    return db.query(queryText, [postId])
+      .then((response) => response.rows);
```

Comment:
I'd suggest this change to be sure you return only one object rather than an array with one object:
`.then((response) => response.rows[0]);`